app.js

```diff
@@ -15,15 +15,15 @@ app.get('/api/articles', getArticles)
 app.post('/api/articles/:article_id/comments', postCommentToArticle)
 
 app.use((err, req, res, next) => {
-    if (err.status === 404) {
+    if (err.status === 404 || err.code === '23503' ) {
```

Comment:
When you can, check the final part of the error handling notes to make this section more robust and easier to read.